hicommonwealth/commonwealth
# Added Skeleton in Discussion + Overview + View Thread page

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/4197

## Description of Changes
Added skeleton UI to view thread, discussion, and overview pages

## Test Plan
- Visit the overview page, and verify it shows a skeleton UI on slow connections (its a little hard to repro this one as the data load is already loaded at this point)
- Visit the discussion page, and verify it shows a skeleton UI on thread cards
- Visit the view thread page, and verify it shows a skeleton UI on page parts

## Deployment Plan
N/A

## Other Considerations
N/A
